config/urls.yaml

```
@@ -0,0 +1,34 @@
+
+# This file contains an array of objects,
+# each having sets of URLs to the equivalent article/video in separate hosting locations.
+# This information is useful when mapping or converting one URL to another,
+# such as when generating a version of an article to be hosted on a specific platform
+
+- section: introduction
+  # URLs of video locations.
+  video:
+    # URL on the innersourcecommons.org site.
+    isc: http://innersourcecommons.org/resources/learningpath/introduction/
+    # URL on the O'Reilly learning plarform.
+    oreilly: https://learning.oreilly.com/videos/introduction-to-innersource/9781492041504/9781492041504-video321606
+    # URL on YouTube.
+    youtube: https://www.youtube.com/watch?v=tP6K5oF2IxQ
+  article:
+    # URL when used as an anchor link in a published PDF.
+    anchor: ''
+    # URL in GitHub.
+    github: https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/master/introduction/01-introduction.asciidoc
+    # URL on the innersourcecommons.org site.
+    isc: ''
+    # URL on the O'Reilly learning plarform.
+    oreilly: ''
+- section: introduction
```

We could add the ordering of the elements here as e.g.
```suggestion
- section: introduction-2
```
(and the same with `introduction-1`)
Good to merge without it, too.

---

I'm thinking of merging without it.  They are all part of the `introduction` section.  The order is already captured by the fact that these are in an array.